package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/monorepo",
-  "version": "5.10.14",
```

Comment:
This should have been `5.10.15`, looks like we forgot to update it last time when doing the release.